macros/models/_sector/nft/nft_mints.sql

```diff
@@ -0,0 +1,185 @@
+{% macro nft_mints(blockchain, base_contracts, base_traces, erc20_transfer, base_transactions, eth_currency_contract ) %}
+
+WITH namespaces AS (
+    SELECT 
+        address
+        , min_by(namespace, created_at) AS namespace
+	FROM {{ base_contracts }}
+	GROUP BY address
+	)
+
+, nfts_per_tx_tmp AS (
+    SELECT 
+        tx_hash
+        , sum(amount) AS nfts_minted_in_tx
+    FROM {{ ref('nft_transfers') }}
+    WHERE 
+        blockchain = '{{blockchain}}'
+        {% if is_incremental() %}
+        AND {{incremental_predicate('block_time')}}
+        {% endif %}
+    GROUP BY tx_hash
+    )
+
+, nfts_per_tx as (
+    SELECT
+        tx_hash
+        , case when nfts_minted_in_tx = UINT256 '0' THEN UINT256 '1' ELSE nfts_minted_in_tx END as nfts_minted_in_tx
+    FROM nfts_per_tx_tmp
```

Comment:
looked into these
```
    select * from (
    SELECT 
        tx_hash
        , sum(amount) AS nfts_minted_in_tx
    FROM nft_ethereum.transfers
    group by tx_hash
    ) where nfts_minted_in_tx = 0
```
they basically all are "mints" that had an ERC1155 transfer with 0 amount (thus not actually minting anything). I assume mostly due to bad contract design that emits the event even if no token was actually minted. 
I think we should filter out these cases instead of adjusting them to 1.